zerver/views/streams.py

```diff
@@ -483,13 +481,14 @@ def update_subscriptions_property(request, user_profile, stream_id=REQ(), proper
                                                   subscription_data=subscription_data)
 
 @has_request_variables
-def update_subscription_properties_backend(request, user_profile, subscription_data=REQ(
-        validator=check_list(
-            check_dict([("stream_id", check_int),
-                        ("property", check_string),
-                        ("value", check_variable_type(
-                            [check_string, check_bool]))])))):
-    # type: (HttpRequest, UserProfile, List[Dict[str, Any]]) -> HttpResponse
+def update_subscription_properties_backend(
+        request: HttpRequest, user_profile: UserProfile,
+        subscription_data: List[Dict[str, Any]]=REQ(
+            validator=check_list(
+                check_dict([("stream_id", check_int),
+                            ("property", check_string),
+                            ("value", check_variable_type(
+                                [check_string, check_bool]))])))) -> HttpResponse:
```

Comment:
Okay, again, this one looks a bit confusing and scary! Maybe do this instead:
```
def update_subscription_properties_backend(
         request: HttpRequest, user_profile: UserProfile,
         subscription_data: List[Dict[str, Any]]=REQ(
             validator=check_list(
                 check_dict([("stream_id", check_int),
                                    ("property", check_string),
                                    ("value", check_variable_type([check_string, check_bool]),])
             )
         )
) -> HttpResponse:
```

Maybe don't copy-paste this since GitHub's indentation seems weird but make it so that its clear which arguments are being passed to which function! :) That makes it easier for other people to add more arguments in the future! :)